azkaban-executor/azkaban-emr-jobtype/src/main/java/uk/gov/dwp/dataworks/azkaban/model/EmrClusterStatus.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.dwp.dataworks.azkaban.model;
+
+public enum EmrClusterStatus {
+    BOOTSTRAPPING(true, false),
+    RUNNING(true, false),
+    STARTING(true, false),
+    TERMINATED(false,true),
+    TERMINATED_WITH_ERRORS(false, true),
+    TERMINATING(true, true),
+    WAITING(true, true);
```

Comment:
Waiting is completed? I'll have to check where this is used in the code...

Reply:
Waiting is only used for clusters like this in dev. I think this is the right setting for waiting. If a cluster is waiting it must be explicitly set to do so after finishing steps, which we would never actually do with this plugin in prod.